src/main/java/myboss/MyBoss.java

```diff
@@ -0,0 +1,105 @@
+package myboss;
+
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents the Personal Assistant Chatbot that helps a person to keep track of various things.
+ * A MyBoss Object corresponds to a Personal Assistant Chatbot.
+ */
+public class MyBoss {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Creates a MyBoss Object with the specified file path for saving tasks.
+     *
+     * @param filePath path to file where tasks will be saved.
+     */
+    public MyBoss(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        tasks = new TaskList(storage.loadTaskListFromFile());
+    }
+
+    /**
+     * Logic of the Personal Assistant Chatbot that receives input and acts accordingly.
+     */
+    public void run() {
+        ui.outputGreeting();
+
+        while (true) {
+            String userCmd = ui.getUserCmd();
+            String[] splitUserCmd = Parser.splitUserCmd(userCmd);
+            String command = splitUserCmd[0];
+
+            String remainingUserInput = Parser.getRemainingUserCmd(splitUserCmd);
+
+            try {
+                switch(command) {
+                    case "bye":
+                        storage.updateFile(tasks.getTaskList());
+                        ui.outputFarewell();
+                        System.exit(0);
+                    case "list":
+                        ui.outputTaskList(tasks.getTaskList());
+                        break;
```

Comment:
Perhaps you can make this switch statement compliant to the Java coding standard (no case indentations)? e.g.
```java
switch (type) {
case "bye":
    break;
case "list":
    break;
...
}
```